Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
@qu1j0t3 Yes, please use `with-xz`. You can still retain the description that makes it clear that this is actually referring to LZMA compression. Maybe one day we'll be able to handle this better in our dependency system, i.e. be able to better separate option names from formula names where it makes sense without all the downsides `depends_on <formula> if <condition>` currently has.